scripts/twemoji-regex-updater.mjs

```diff
@@ -0,0 +1,71 @@
+import { fetch, FetchResultTypes } from '@sapphire/fetch';
+import { Time, Timestamp } from '@sapphire/time-utilities';
+import { green, red, yellow } from 'colorette';
+import { rm, writeFile } from 'node:fs/promises';
+import { URL } from 'node:url';
+
+async function importFileFromWeb({ url, temporaryFileName }) {
+	const body = await fetch(url, FetchResultTypes.Text);
+
+	const temporaryOutputFile = new URL(temporaryFileName, import.meta.url);
+
+	await writeFile(temporaryOutputFile, body);
+	const loadedModule = await import(temporaryOutputFile);
+
+	await rm(temporaryOutputFile);
+
+	return loadedModule;
+}
+
+const filePrefix = [
+	'/**',
+	' * Regex that can capture a Twemoji (Twitter Emoji)',
+	' * @raw {@linkplain https://github.com/twitter/twemoji-parser/blob/master/src/lib/regex.js See official source code}',
+	' */',
+	'export const TwemojiRegex =',
+	'\t'
+].join('\n');
+const fileSuffix = [';', ''].join('\n');
+
+const shaTrackerFileUrl = new URL('sha-tracker.json', import.meta.url);
+const twemojiRegexFileUrl = new URL('../packages/discord-utilities/src/lib/TwemojiRegex.ts', import.meta.url);
+const oneMonthAgo = Date.now() - Time.Month * 6;
+const timestamp = new Timestamp('YYYY-MM-DD[T]HH:mm:ssZ').display(oneMonthAgo);
+
+const url = new URL('https://api.github.com/repos/twitter/twemoji-parser/commits');
+url.searchParams.append('path', 'src/lib/regex.js');
+url.searchParams.append('since', timestamp);
+
+const [commits, { default: ciData }] = await Promise.all([
+	fetch(url, FetchResultTypes.JSON), //
+	import(shaTrackerFileUrl, { assert: { type: 'json' } }) //
+]);
+
+const data = { sha: commits.length ? commits[0].sha : null, length: commits.length };
+if (!data) {
```

Comment:
Never not true

Reply:
weird must be an artifact from an earlier write up in gqlp pokemon